src/components/Details/DetailObject.tsx

```diff
@@ -104,7 +105,9 @@ class DetailObject extends React.Component<DetailObjectProps> {
         ) : (
           undefined
         )}
-        <ul style={{ listStyleType: 'none' }}>{childrenList}</ul>
+        <ul className={'details'} style={{ listStyleType: 'none' }}>
```

Comment:
Probably, `style={{ listStyleType: 'none' }}` deserves to be moved to `DetailObject.scss`.
(Just a comment; feel free to ignore)